tendrl/commons/message.py

```diff
@@ -50,6 +50,8 @@ def __init__(self, priority, publisher, payload, job_id=None,
         else:
             self.caller = caller
         self.priority = priority
+        if not publisher:
+            publisher = NS.publisher_id
```

Comment:
Well I see no problem in a class member function validating values of all the class data members specially when you say a utility outside the class can do it....